docs/sources/dashboards/share-dashboards-panels/shared-dashboards/index.md

```diff
@@ -77,11 +77,19 @@ This feature was previously called **email sharing**.
 Available in [Grafana Enterprise](ref:grafana-enterprise) and [Grafana Cloud](/docs/grafana-cloud).
 {{< /admonition >}}
 
-{{< docs/private-preview product="Sharing externally with specific people" >}}
+<!-- {{< docs/private-preview product="Sharing externally with specific people" >}}-->
 
-To share a dashboard with specific external users, you can send them a link by email. Use this option when you only want to share your dashboard with specific people. When you share dashboards by email, recipients receive a one-time use link that's valid for **one hour**. Once the link is used, the viewer has access to the shared dashboard for **30 days**.
+{{< admonition type="note" >}}
+Sharing externally with specific people is currently in [private preview](https://grafana.com/docs/release-life-cycle/#private-preview).
+
+This feature will have a cost by active users after being promoted into general availability.
```

Comment:
Do we need to specify how the feature will be a paid feature (i.e. cost by active user) or can we just say this feature will be a paid or premium paid feature once it is released

Also will we grandfather early adopters of this to always have this functionality as free? Might be a little annoying to have a feature for free and then have to start paying for it 😬

Reply:
> Do we need to specify how the feature will be a paid feature (i.e. cost by active user) or can we just say this feature will be a paid or premium paid feature once it is released

I'll leave that up to @thanos-karachalios 

> Also will we grandfather early adopters of this to always have this functionality as free? Might be a little annoying to have a feature for free and then have to start paying for it 😬

We've had a note in the docs about an eventual cost for this feature since v9.5. We're just making the note be future oriented here to avoid confusion, since we're still in preview on it. But ultimately, that's outside the scope of this PR, and something you and @thanos-karachalios can discuss.

Reply:
This added information is useful since we are communicating the status of the feature (private preview) and the incurred cost once the feature will be on GA. I would keep this as is, but in addition to the above.